odoo/tests/common.py

```diff
@@ -1842,6 +1850,12 @@ def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=N
                 for name, value in cookies.items():
                     self.browser.set_cookie(name, value, '/', HOST)
 
+            if cpu_throttling > 1:
+                timeout *= cpu_throttling  # extend the timeout as test will be slower to execute
+                _logger.warning('CPU throttling mode is only suitable for local testing - ' \
```

Comment:
This warning will make the build fail on runbot

Reply:
should I change the level to info or disable the warning only if the os variable is used (which we will assume is the dedicated build on runbot to test it) ?

Reply:
Disable the warning in case of os variable usage is the preferred solution.

Reply:
what do you think of the last change ( https://github.com/odoo/odoo/compare/369b5bde9812ec042d60544b939505450857263f..d92f53f75b8ce32db88a8c6f80f83856dbd527af )
basically swtiching to info if the os parameter is used (as its valuable information to reproduce in my opinion) and keeping in warning when used as a parameter